src/renderers/webgl/WebGLBufferRenderer.js

```diff
@@ -4,6 +4,8 @@
 
 function WebGLBufferRenderer( gl, extensions, info ) {
 
+	var isWebGL2 = typeof WebGL2RenderingContext !== 'undefined' && gl instanceof WebGL2RenderingContext;
```

Comment:
There're duplicated this webgl2 context check line across modules.

Do you think it'd be worth to have `isWebGL2` function in `utils.js` or somewhere?

    function isWebGL2( gl ) {

        return typeof WebGL2RenderingContext !== 'undefined' && gl instanceof WebGL2RenderingContext;

    }

Reply:
Can you determine that when the context is instantiated and save it as a property?
```javascript
gl.isWebGL2 = boolean;
```

Reply:
I think we can if we don't mind polluting the context object.

Reply:
Updated, Thanks!